src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Card, Text, Title, Button } from '@gnosis.pm/safe-react-components'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+type Props = {
+  error: Error
+  componentStack: string
+  resetError: () => void
+}
+
+const GlobalErrorBoundaryFallback: React.FC<Props> = ({ error, componentStack }: Props) => {
```

Comment:
React.FC adds an implicit definition for `children`. I thought we agreed on using React.ReactElement as a return type

Reply:
good catch, I forget to remove React.FC type, I was testing something different. Yep, you are right with React.ReactElement.